style.css

```diff
@@ -0,0 +1,107 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&family=Ubuntu:wght@300&display=swap");
+
+/**
+*! Color variables
+**/
+
+/**
+*! Removing default browser style
+**/
+* {
+  box-sizing: border-box;
+}
+
+* body {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+}
+
+* a {
+  text-decoration: none;
+}
+
+* li {
+  list-style: none;
+}
+
+/**
+*! Header styles inputs here
+**/
+.navbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 1.125rem;
+  line-height: 1.25rem;
+  margin-left: 1rem;
+  color: #6070ff;
+}
+
+[class^=bar] {
+  display: block;
+  background-color: #6070ff;
+  width: 1.875rem;
+  height: 0.1875rem;
+  margin: 0.3125rem;
+  margin-left: 14rem;
+}
+
+/**
+*! Main section styles inputs here
+**/
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url("./images/header-shapes mobile.svg");
+  background-repeat: no-repeat;
+  background-position: top left;
```

Comment:
- [ ] This background image is not compatible with different mobile screen sizes, kindly download each piece of the image alone from the Figma design and then try to set each piece to its own position in your headline.

### Your design 
![image](https://user-images.githubusercontent.com/68030297/219109976-395f1af0-deb1-4e63-8f2d-e2e0d7af8a01.png)

## The required Design
![image](https://user-images.githubusercontent.com/68030297/219110144-85e5c2bf-896f-4e32-a925-8ccccd553b5e.png)